arm/Microsoft.Web/staticSites/.parameters/github.parameters.json

```diff
@@ -0,0 +1,76 @@
+{
```

Comment:
I suggest renaming this to `parameters.json` as `github` in the name seems a bit specific.

Reply:
Github is one of the many options you have on this. I have seen GitHub, DevOps and Other being main types. The solution interacts with the repo+ci/cd solution to create and regiser a pipeline file stored in the repo.

Reply:
Reduced the number of testcases to empty minimum (min.parameters.json) and extended scenario (parameters.json).